src/App.js

```diff
@@ -1,34 +1,62 @@
 import './App.css';
 import {useEffect, useState} from 'react';
 
-import Footer from './components/Footer/Footer';
 import Header from './components/Header/Header';
+import Movie from './components/Movie/Movie';
 
 function App() {
-  const API_KEY = process.env.REACT_APP_API_KEY; ////// check back!
+  const API_KEY = process.env.REACT_APP_API_KEY;
   const url = 'https://api.themoviedb.org/3/movie/';
 
   const [moviesData, setMoviesData] = useState([]);
+  const [error, setError] = useState(null);
+
+  async function fetchMovieData() {
+    setError(null);
+    try {
+      const response = await fetch(`${url}popular?api_key=${API_KEY}`);
+
+      if (response.ok) {
+        const fetchedMovieData = await response.json();
+        setMoviesData(fetchedMovieData.results);
+      } else {
+        setError(new Error());
+      }
+    } catch (error) {
+      setError(new Error('Sorry Data can not be fetched'));
+    }
+  }
 
   useEffect(() => {
-    fetch(`${url}popular?api_key=${API_KEY}`)
-      .then(response => response.json())
-      .then(fetchData => {
-        setMoviesData(fetchData.results);
-        console.log(fetchData);
-      });
-  }, []);
+    fetchMovieData();
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
 
   return (
     <div className="main-app">
+      {}
       <Header />
-      <div className="movie__container">
-        {moviesData.map(movie => (
-          <div className="movie__single" key={movie.id}>
-            <h3 className="movie__title">{movie.title}</h3>
-          </div>
-        ))}
-      </div>
+      <main>
+        <div className="movie__container">
+          {error && (
+            <div>
+              <h4 style={{color: '#3083dc'}}>Ooooops... Something went wrong.</h4>
```

Comment:
If you do not have an `h1`-`h3` in your document, you should not use an `h4` here.